curriculum/dictionaries/italian/comments.json

```diff
@@ -1,110 +0,0 @@
-{
```

Comment:
Tom... are you trying to get rid of all the Italians' work again...?

Reply:
umm, what - how did this get here